101-maps-create/azuredeploy.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name":{
+            "type": "string",
+            "defaultValue": "exampleMap",
+            "metadata": {
+                "description": "The name for your Azure Maps account."
+            }
+        },
+        "pricingTier": {
+            "type": "string",
+            "allowedValues":[
+                "S0",
+                "S1"
+            ],
+            "defaultValue": "S0",
+            "metadata": {
+                "description": "The pricing tier for the account. Use S0 for small-scale development. Use S1 for large-scale applications."
+            }
+        }
+    },
+    "variables": {
+    },
+    "resources": [
+        {
+            "name": "[parameters('account_name')]",
+            "type": "Microsoft.Maps/accounts",
+            "apiVersion": "2020-02-01-preview",
+            "location": "global",
+            "tags": {
+            },
+            "sku": {
+                "name": "[parameters('pricingTier')]"
+            }
+        }
+    ],
+    "outputs": {
+        "primaryKey": {
```

Comment:
remove this - you're not allowed to output secrets.

Reply:
This is used to show that the key can be captured and used in additional deployment steps. Can there be an exception?

Reply:
No, unfortunately (or maybe it's a good thing) customers tend to use these examples in production environments and adopt the bad practice.  If there is a use case for consuming the key, you can always add that to the sample or another one.